components/base/Collapsible.tsx

```diff
@@ -60,6 +48,23 @@ const ImageStyles = css`
   width: 24px;
 `;
 
+const InnerContentWrapper = styled.div`
+  margin: 0px 70px 25px 30px;
+  * {
+    margin: unset;
+  }
+  color: ${PARAGRAPH_TEXT};
+  font-family: ${ROBOTO};
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 28px;
+
+  &.visuallyhidden {
+    transition-delay: 200ms;
+    visibility: hidden;
```

Comment:
Added a transition to the make the inner content visible while the Collapsible is **closing**.